cmd/livepeer_cli/wizard_transcoder.go

```
@@ -38,11 +39,21 @@ func (w *wizard) promptOrchestratorConfig() (float64, float64, int, int, string)
 		feeShare       float64
 	)
 
-	fmt.Printf("Enter block reward cut percentage (default: 10) - ")
-	blockRewardCut = w.readDefaultFloat(10.0)
+	orch, _, err := w.getOrchestratorInfo()
+	if err != nil || orch == nil {
+		fmt.Println("unable to get current reward cut and fee share")
+		blockRewardCut = 0
+		feeShare = 0
+	}
+
+	blockRewardCut = eth.ToPerc(orch.RewardCut)
+	feeShare = eth.ToPerc(orch.FeeShare)
+
+	fmt.Printf("Enter block reward cut percentage (current=%v default=10) - ", blockRewardCut)
+	blockRewardCut = w.readDefaultFloat(blockRewardCut)
 
-	fmt.Printf("Enter fee share percentage (default: 5) - ")
-	feeShare = w.readDefaultFloat(5.0)
+	fmt.Printf("Enter fee share percentage (current=%v default=5) - ", feeShare)
+	feeShare = w.readDefaultFloat(eth.ToPerc(orch.FeeShare))
```

`readDefaultFloat` doesn't provide any feedback in case there was error parsing number from string.
And current code doesn't show to the user what actual value it will use for the configuration. So if user makes typo default value will be silently used.

---

didn't really consider this in scope for this PR but we can add it if you want

---

np, lets keep this PR small